<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsdialog</class>
 <widget class="QDialog" name="settingsdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>479</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="applyPushButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="parameterGroupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>241</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameter</string>
   </property>
   <widget class="QComboBox" name="baudRateComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dataBitsComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="parityComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="stopBitsComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="flowControlComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="baudRateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>BaudRate</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataBitsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Data bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="parityLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity</string>
    </property>
   </widget>
   <widget class="QLabel" name="stopBitsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="flowControlLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow control</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="selectSerialPortGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Serial Port</string>
   </property>
   <widget class="QLabel" name="comPortlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Com Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comPortComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="descriptionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="manufacturerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Manufacturer</string>
    </property>
   </widget>
   <widget class="QLabel" name="serialNumberLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial number</string>
    </property>
   </widget>
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="vendorIdLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vendor ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="productIdLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Product ID</string>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="localEchoRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Local echo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
